tests/unit/plugins/framework_specific/airflow/__snapshots__/test_writer_airflow.ambr

```diff
@@ -630,9 +630,9 @@
   from airflow.utils.dates import days_ago
   
   
-  def task_a0():
+  def task_b0():
```

Comment:
I am a little concerned here as the variables seem to have flipped. Maybe the `sorted` command was important in `airflow_pipeline_writer.py`, and if so, would `sorted` also be required in the other pipeline writers as well?

Reply:
See https://github.com/LineaLabs/lineapy/pull/876#discussion_r1085784728